setup/install/providers/kubernetes-v2/gke/index.md

```diff
@@ -0,0 +1,61 @@
+---
+Layout: single
+title:  "Google Kubernetes Engine (GKE) Setup"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[GKE](https://cloud.google.com/kubernetes-engine/) that Spinnaker can
+manage and deploy workloads to. The process is very simple, but some
+considerations need to be made when configuring how Spinnaker will authenticate
+against your cluster.
```

Comment:
This page describes how to set up a Kubernetes cluster on GKE to be used as a Spinnaker provider. The process is very simple, but you need to do some specific things to allow Spinnaker to authenticate against your cluster.